var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -99,6 +99,7 @@ class Gdal(AutotoolsPackage):
     variant('cryptopp',  default=False, description='Include cryptopp support')
     variant('crypto',    default=False, description='Include crypto (from openssl) support')
     variant('grib',      default=False, description='Include GRIB support')
+    variant('64bitIO',   default=True,  description='Enable 64-bit file I/O')
```

Comment:
Just learned that variants should be all lowercase. Any suggestions that look better than `64bitio`?

Reply:
```suggestion
    variant('unix_stdio_64',   default=True,  description='Utilize 64 stdio api')
```
Let's match the name and description from `configure`

Reply:
Should we change the description too?

Reply:
Done.